apiary.apib

```diff
@@ -8113,11 +8113,13 @@ Fetch cloudPlatform url for type and id
         + product_id: `d905ff57-e866-0f59-9d1e-1fd4538bfae1` (string, optional)
 
 ## NoteSubjectTypes (enum)
-+ `contact`
 + `company`
++ `contact`
++ `creditNote`
 + `deal`
 + `invoice`
-+ `creditNote`
-+ `subscription`
++ `order`
```

Comment:
`order` is not an API v2 concept yet. 
No need to include it if we have no way to find UUIDs with that note subject type.